### PR TITLE
Merch item enabled

### LINKED ISSUE #80: User Story 52, Merchant enables an item
```
As a merchant
When I visit my items page
And I click on an "enable" button or link for an item
I am returned to my items page
I see a flash message indicating this item is now available for sale
I see the item is now enabled
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #83

depends on #82

### PR DESCRIPTION
**Pull request by stoic-plus**
closes #80 and #81 

* adds flash messages to enabling/disabling items 
* adds stuff from `item_params` with thumbnail

### CODE REVIEW

File: app/views/dashboard/items/index.html.erb [hunk 1]
```diff
@@ -16,7 +16,11 @@
             <tr id=<%="item-#{item.id}"%>>
                 <td><%= item.id%></td>
                 <td><%= item.name%></td>
-                <td><%= image_tag(item.thumbnail, id:("#{item.thumbnail}")) %></td>
+                <% begin %>
+                  <td><%= image_tag(item.thumbnail, id:("#{item.thumbnail}")) %></td>
+                <% rescue %>
+                  <td><%= image_tag("no_img", id:("no_img")) %></td>
```
Comment (nicovigil1) [inline app/views/dashboard/items/index.html.erb]: needs to be "no_img.jpg"

File: spec/features/users/merchant/merchant_create_spec.rb [hunk 2]
```diff
@@ -9,11 +9,9 @@
       visit dashboard_items_path
 
       click_on 'add new item'
-      save_and_open_page
```
Comment (nicovigil1) [inline spec/features/users/merchant/merchant_create_spec.rb]: whoops. good catch

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -195,9 +195,10 @@
         user2 = create(:user)
         merchant = create(:user, role: 1)
 
-        expected = User.default
-        # FIXME
-        expect(expected).to eq([user1, user2])
+        defaults = User.default
+        expect(defaults.size).to eq(2)
+        expect(defaults.first.id).to eq(user1.id)
```
Comment (nicovigil1) [inline spec/models/user_spec.rb]: this breaks occasionally - not sure if it's order related. Just a heads up

File: spec/features/users/merchant/merchant_can_enable-disable_items_spec.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      @m = create(:user, role: 1)
+      @i_1 = create(:item, user: @m, enabled: true)
+      @i_2 = create(:item, user: @m)
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@m)
+      visit dashboard_items_path
+    end
+
+    it 'shows button to enable or disable and clicking redirects back to dashboard_items' do
+      expect(page).to have_button("Enable", count: 1)
```
Comment (nicovigil1) [inline spec/features/users/merchant/merchant_can_enable-disable_items_spec.rb]: count?

File: app/views/dashboard/items/index.html.erb (hunk 1 above)
Comment (tonyr729) [reply app/views/dashboard/items/index.html.erb]: No, it should be just `no_img`. Check documentation for `image_tag`.
